recipes/daw_header_libraries/all/conandata.yml

```diff
@@ -17,24 +20,6 @@ sources:
   "2.68.1":
     url: "https://github.com/beached/header_libraries/archive/v2.68.1.tar.gz"
     sha256: "51bdb042959373729009f91449c492f58bb63262146463a767f17d3de6fb2687"
-  "2.65.1":
```

Comment:
We typically leave older versions unless there's a technical reason to drop them -- is there a reason here?

Reply:
@prince-chrismc 
I have read the following document and try to erase the old version to save CI time.
If there is any inconvenience, I will cancel the deletion.

https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/reviewing.md#supported-versions

Reply:
Try not to remove them unless theres a problem -- this is a header only it's takes nothing compared to the culprits like boost or cmake...

This time it's fine but lets not remove them without a really good reason